core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/AmbariRepositoryVersionService.java

```diff
@@ -69,17 +67,17 @@ public String getRepositoryVersion(long clusterId, Orchestrator orchestrator) th
         return result;
     }
 
-    public void setBaseRepoURL(Long stackId, long clusterId, Orchestrator orchestrator, StackService ambariClient) throws CloudbreakException {
+    public void setBaseRepoURL(long clusterId, Orchestrator orchestrator, StackService ambariClient) throws CloudbreakException {
         StackRepoDetails stackRepoDetails = getStackRepoDetails(clusterId, orchestrator);
         if (stackRepoDetails != null) {
             try {
                 LOGGER.info("Use specific Ambari repository: {}", stackRepoDetails);
                 AmbariRepo ambariRepoDetails = clusterComponentConfigProvider.getAmbariRepo(clusterId);
                 long ambariVersion = extractAmbariVersion(ambariRepoDetails);
```

Comment:
extractAmbariVersion method does not makes sense e.g. it does not give a valid value for 2.5.11.0 since 2.5.11.0 is consdered to be newer than even 3.0.0.0